src/libstd/sys/common/net2.rs

```diff
@@ -127,6 +128,50 @@ pub fn lookup_host(host: &str) -> io::Result<LookupHost> {
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+// lookup_addr
+////////////////////////////////////////////////////////////////////////////////
+
+extern "system" {
+    fn getnameinfo(sa: *const libc::sockaddr, salen: socklen_t,
+        host: *mut c_char, hostlen: libc::size_t,
```

Comment:
Stylistically we typically line up with this argument with the `sa` argument (align on the parens)